init.sh

```diff
@@ -9,16 +9,16 @@ channel_display_name="AI"
 user_name="root"
 user_password="$(openssl rand -base64 14)"
 
-echo "Starting Mattermost and localai for demo..."
-
 if [ "$backend" == 'localai' ]; then
+	echo "Starting Mattermost and localai for demo..."
 	docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
 else
+	echo "Starting Mattermost and openai for demo..."
 	docker-compose -f docker-compose.yml up -d 
 fi
 
 echo "Mattermost is starting. Waiting 35 seconds."
-sleep 35
+#sleep 35
```

Comment:
Do we not need this?

Reply:
We do. It was turned off for testing. Fixed now.